doc/release-notes/0.11/0.11.md

```diff
@@ -0,0 +1,176 @@
+<td valign="top">OpenJDK8 and later (All platforms)</td>
+<td valign="top">`-Xshareclasses:bootClassesOnly` prevents non-bootstrap class loaders caching classes in the shared classes cache. `-Xshareclasses:fatal` prevents the shared classes cache being started if an error occurs. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3184">#3184</a></td>
+<td valign="top">VM container-awareness is now enabled by default</td>
+<td valign="top">OpenJDK8 and later (Linux)</td>
+<td valign="top">In earlier releases, this behavior was enabled by setting the `-XX:+UseContainerSupport` option. This setting is now the default. When the VM is running in a container, and a memory limit is set, the VM allocates more memory to the Java heap. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3054">#3054</a></td>
+<td valign="top">Concurrent scavenge mode is now available on x86</td>
+<td valign="top">OpenJDK8 and later (x86-64: Linux and Windows only)</td>
+<td valign="top">When this mode is enabled, (`-Xgc:concurrentScavenge`) the VM attempts to reduce GC pause-times for response-time sensitive, large heap applications. This mode is available only on compressed references builds. Attempting to use this option on a non-compressed references build results in an *unrecognized option* error message. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3062">#3062</a></td>
+<td valign="top">New command line option `-XX:[+|-]PositiveIdentityHash`</td>
+<td valign="top">OpenJDK8 and later (All platforms)</td>
+<td valign="top">When enabled, identity hash codes (`System.identityHashCode` / `Object.hashCode`) always return non-negative values. Because the performance of identity hash-intensive operations might be impacted, this option is not enabled by default.</td>
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.11.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support</td>
+<td valign="top">AIX and macOS</td>
+<td valign="top">Inability to diagnose problems with the VM, garbage collector, or JIT.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/129">#129</a></td>
+<td valign="top">OpenSSL support</td>
+<td valign="top">AIX</td>
+<td valign="top">Currently, builds of OpenJDK 8 with OpenJ9 for AIX do not bundle the OpenSSL v1.1.1 support due to an unresolved problem.</td>
```

Comment:
This statement applies to builds at AdoptOpenJDK. A user creating a build can do what they like.